index.html

```diff
@@ -70,17 +70,76 @@
   homeButton.options.position = 'topright'
   homeButton.addTo(map)
 
-  L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
-      attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
-    }).addTo(map);
+  // Look if world coastline map ist locally available and use it
+  var tilemapUrl = false;
+  var tilemapMeta = false;
+  $.ajax({
+    // path of list of maps in Signal K server
+    url: "/signalk/v1/api/resources/charts",
+    async: false,
+    success: function(data) {
+      // Not the data we expected? URL wrong?
+      if (data === null) {
+        console.log("Signal K server not found.")
+        return;
+      }
+      if (typeof data !== 'object') {
+        console.log("Signal K server not found.")
+        return;
+      }
+
+      // Is the world coastline map installed?
+      if (typeof data['world-coastline.mbtiles'] !== 'undefined') {
+        // this will be "/charts/world-coastline.mbtiles/{z}/{x}/{y}"
+        console.log("Show the world coastline map.")
+        tilemapUrl = data['world-coastline.mbtiles'].tilemapUrl;
+        tilemapMeta = {
+          attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
+          maxZoom: 11,
+          tileSize: 1024,
+          zoomOffset: -2,
+        };
+        return;
+      }
+      // Is any other map installed on Signal K server?
+      if(!$.isEmptyObject(data)) {
+        // Take one map we randomly find in list of maps
+        $.each(data, function(index, firstMap) {
+          console.log("Map used: ", firstMap.identifier);
+          tilemapUrl = firstMap.tilemapUrl;
+          tilemapMeta = {
+            attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
+            maxZoom: firstMap.maxzoom,
+          };
+          // exit each loop
+          return false;
+        });
+        return;
+      }
+    }
+  });
+
+  // Fallback to Internet tiles
+  if (!tilemapUrl) {
+    tilemapUrl = "http://{s}.tile.osm.org/{z}/{x}/{y}.png";
+    tilemapMeta = {
+      attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
+      maxZoom: 13,
+    };
+  }
 
+  L.tileLayer(tilemapUrl, tilemapMeta).addTo(map);
+
+
+  /*
+  // Add a strange Finnish layer
```

Comment:
Could you please remove this funny finnish layer? Local curiosity, your PR makes it obsolete - if you want one, add one to your server. 

Actually we could use a plugin that would allow you to add 3rd party charts off the Internet to the server, now you would have to add those to defaults.json.